Automattic/jetpack
# Calypsoify: Copy the module code to the Calypsoify package

Resolves https://github.com/Automattic/vulcan/issues/319

## Proposed changes:

* Copy the Calypsoify code from the Jetpack module into the newly introduced Calypsoify PHP package.
* Minor adjustments - adding namespace, removing unnecessary mods.js file, moving JS files to JS folder and CSS to a CSS folder, plus readme tweaks.
* Additionally, updated how the Jetpack version constant is retrieved and re-added the phan baseline file due to the remaining phan test results (the rest appear to be false positives or issues that previously existed, though I'll be able to clarify any issues when actually hooking everything up).

### Other information:


## Jetpack product discussion

- https://github.com/Automattic/vulcan/issues/319
- pfwV0U-8E-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* This package is not connected to anything yet so a generic review that everything appears to be in place should be enough. More will likely be needed when the package is required, but that will come later.
* That said, the build can be tested by checking out this PR and running `jetpack build packages/calypsoify --production`. You should see a 'build' directory created with 3 files - the minified JS, CSS and a RTL CSS file.
